resources/Scripts/story.json

```diff
@@ -0,0 +1,18 @@
+{
+  "inkVersion": 17,
+  "root": [
+    "^Welcome to NovelRT.",
+    "\n",
+    "^This text is coming from Ink.",
+    "\n",
+    "^Each click should return one line.",
+    "\n",
+    "^And then it should loop back to the beginning.",
+    "\n",
+    "done",
+    {
+      "#f": 3
+    }
+  ],
+  "listDefs": {}
+}
```

Comment:
Any chance we could push up the ink source for this?

Reply:
You mean the raw ink file? I don't see a "lot" of benefit given that you can import/export the json into inkle.